packages/java-edition/src/mcfunction/mcdocAttributes.ts

```diff
@@ -32,11 +46,20 @@ const scoreHolderValidator = validator.alternatives<ScoreHolderConfig>(
 )
 
 export function registerMcdocAttributes(meta: core.MetaRegistry, rootTreeNode: mcf.RootTreeNode) {
-	mcdoc.runtime.registerAttribute(meta, 'command', () => undefined, {
-		// TODO: validate slash
+	mcdoc.runtime.registerAttribute(meta, 'command', commandValidator, {
 		// TODO: fix completer inside commands
-		stringParser: () => {
-			return mcf.command(rootTreeNode, parser.argument)
+		stringParser: ({ slash, max_length, empty }) => {
+			return (src, ctx) => {
+				if ((empty && !src.canRead()) || (slash === 'chat' && src.peek() !== '/')) {
+					return core.string({
+						unquotable: { blockList: new Set(), allowEmpty: true },
+					})(src, ctx)
+				}
+				return mcf.command(rootTreeNode, parser.argument, {
+					slash: slash === 'chat' ? 'allowed' : slash,
```

Comment:
If it's chat as in a /tellraw component, a slash is actually required (without the slash, it used to just send a chat message in your name, but I think since the chat signing thing, it's just disallowed).

On signs however, a slash is optional.

If the aim is not to 100% solve these cases, the linked issue as wrong, as it describes a v3 bug where DPH+ would show an error when omitting the slash on a sign.

Reply:
Hmm so on signs in the `run_command` action, it will run the command when you don't start with a slash?